src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+// import java.io.;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final byte OPERATION_TYPE = 0;
+    private static final byte AMMOUNT = 1;
+    private int supply;
+    private int buy;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder resultStringBuilder = new StringBuilder();
+        readFile(fromFileName);
+        resultStringBuilder.append("supply,").append(supply).append(System.lineSeparator())
+                .append("buy,").append(buy).append(System.lineSeparator())
+                .append("result,").append(supply - buy).append(System.lineSeparator());
```

Comment:
lets move logic of creating report in separate file

Reply:
Do you mean to create a separate method?

Reply:
Perhaps it would be logical to move this code to dataProcessing?